packages/financial-templates-lib/src/helpers/acrossFeesCalculator.ts

```diff
@@ -0,0 +1,89 @@
+// This Util calculates the across realized LP fees. See https://gist.github.com/chrismaree/a713725e4fe96c531c42ed7b629d4a85
+// gist for a python implementation of the logic in this file.
+// TODO: add a link to chase's notebooks once they are production ready to explain how the logic works past the comments included here.
+
+import BN from "bn.js";
+import Decimal from "decimal.js";
+import Web3 from "web3";
+
+const { toBN, toWei, fromWei } = Web3.utils;
+const toBNWei = (number: string | number) => toBN(toWei(number.toString()).toString());
+const fixedPointAdjustment = toBNWei("1");
+
+export interface RateModel {
+  UBar: BN; // denote the utilization kink where the slope of the interest rate model changes.
```

Comment:
The `kink` in this case is a value on the "x axis"? Might be worth clarifying, since just describing it as a kink isn't that useful

Reply:
added more detials